app/views/layouts/application.html.erb

```diff
@@ -24,7 +24,7 @@
       <p><%= t('.language_options') %></p>
       <ul>
         <li><%= link_to 'English', :locale => :en %></li>
-        <li><%= link_to 'Bro', :locale => :bro %></li>
+        <li><%= link_to 'Bro', :pseudo => :pseudo %></li>
```

Comment:
This should be
 \<li><%= link_to 'Pseudo', :locale => :pseudo %>\</li> 
otherwise the link doesn't switch the locale and the link text still displays as 'Bro' instead of 'Pseudo'. You should also have a link to Spanish
 \<li><%= link_to 'Español', :locale => :es %>\</li>